shell/platform/darwin/ios/platform_view_ios.mm

```diff
@@ -313,6 +310,18 @@ bool UpdateStorageSizeIfNecessary() {
   engine_->RunFromFile(main, packages, assets_directory);
 }
 
+ftl::WeakPtr<PlatformViewIOS> PlatformViewIOS::GetWeakPtr() {
+  return weak_factory_.GetWeakPtr();
+}
+
+void PlatformViewIOS::UpdateSurfaceSize() {
+  blink::Threads::Gpu()->PostTask([weak_this = GetWeakPtr()]() {
```

Comment:
Serves them right. Fixed. Rebased.

I myself am partial to `thiz`. That confuses everyone.